src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -87,11 +86,6 @@ public static ServiceProviderWrapper CreateStaticServices(this JobHostConfigurat
             services.AddService(blobWrittenWatcherAccessor);
             ISharedContextProvider sharedContextProvider = new SharedContextProvider();
 
-            // Create a wrapper TraceWriter that delegates to both the user 
-            // TraceWriters specified via config (if present), as well as to Console
-            TraceWriter trace = new ConsoleTraceWriter(config.Tracing, consoleProvider.Out);
```

Comment:
Verify that user traces are still written to console

Reply:
Yep, they are -- they go through the normal ConsoleLogger.